libi2pd/RouterInfo.h

```diff
@@ -120,6 +120,7 @@ namespace data
 			RouterInfo (const std::string& fullPath);
 			RouterInfo (const RouterInfo& ) = default;
 			RouterInfo& operator=(const RouterInfo& ) = default;
+			RouterInfo (const uint8_t * buf, int len, bool);
```

Comment:
this should be

    RouterInfo(const uint8_t * buf, int len, bool checkSignature=true);